h2o-py/h2o/scoring.py

```diff
@@ -0,0 +1,44 @@
+def make_leaderboard(object, leaderboard_frame=None,
+                     sort_metric="AUTO",
+                     extra_columns=[],
+                     scoring_data="AUTO"):
+    from .utils.typechecks import is_type
+    from .frame import H2OFrame
+    from .expr import ExprNode
+    from .exceptions import H2OValueError
```

Comment:
do you get import errors when doing this on top of the file?
We often do this to solve dependency loop issues, but imo it should be avoided when not necessary: better fail early at load time and solve those dependency issues by correct module organization rather than postponing potential issues at runtime.